packages/onboarding/src/navigator/navigator-screens/hooks/use-navigator-buttons.tsx

```diff
@@ -0,0 +1,32 @@
+import { NavigationButtonAsItem } from '../../navigator-buttons';
+import NavigatorItemGroup from '../../navigator-item-group';
+import type { NavigatorScreenObject } from '../types';
+
+const useNavigatorButtons = ( screens: NavigatorScreenObject[] ) => {
+	if ( screens.length === 0 ) {
+		return null;
+	}
+
+	if ( screens.length === 1 ) {
+		return screens[ 0 ].content;
```

Comment:
If there is only 1 screen, we don't need to the navigator buttons. So, return the content of the screen directly. Or should leave the parent component to determine this logic 🤔